website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,132 @@
+
+#### Filter bar
+
+Organizing filters in a  **filter bar** is a common positioning method and consists of one or more dropdowns, buttons, or input components that are used to select values from a set of parameters. A [Segmented Group](/components/segmented-group) can be used within a filter bar to group similar parameters and support complex filtering.
+
+![Filter bar example](/assets/patterns/filter-patterns/filter-bar-segmented-group.png)
+
+**Ex 1.0: Horizontal filter bar**
+
+![Horizontal orientation of the filter bar](/assets/patterns/filter-patterns/layout-filter-bar.png)
+
+#### Sidebar
+
+If space permits, filters can be positioned vertically in a **sidebar**, either at the page level, or in-line with the data set.
+
+- **Page-level sidebar:** can be used for more complex data sets and if filtering contents of the entire page is required.
+- **In-line sidebars:** can be used when the number of filterable parameters is minimal and if space permits.
+
+**Ex 1.1 Page-level filter sidebar**
+
+![Vertical orientation of the filter bar](/assets/patterns/filter-patterns/layout-sidebar-left-page-level.png)
+
+**Ex 1.2 In-line filter sidebar**
+
+![Vertical orientation of the filter bar on the left](/assets/patterns/filter-patterns/layout-sidebar-left-inline.png)
+
+![Vertical orientation of the filter bar on the right](/assets/patterns/filter-patterns/layout-sidebar-right-inline.png)
+
+### Applied filters
+
+The **applied filters** displays which filter values have been applied (if any) and provides a user with a method to clear or reset all filters at once. In most circumstances the filters that have been applied should be represented using a [Tag](/components/tag) component which allows for the individual dismissal of a filter value.
+
+![Applied filters](/assets/patterns/filter-patterns/applied-filters.png)
+
+#### Positioning
+
+Applied filters should be positioned as close as possible relative to the data set to help the user understand what has been filtered upon. In practice this is most often directly above the data set.
+
+**Ex 2.0: Between the filter bar and the data set**
+
+![Applied filters with filter bar positioning](/assets/patterns/filter-patterns/applied-filters-positioning-filter-bar.png)
+
+**Ex 2.1: Above the data set with filters positioned in a sidebar**
+
+![Applied filters with filter sidebar](/assets/patterns/filter-patterns/applied-filters-positioning-sidebar.png)
+
+If the filters are positioned in a page-level sidebar the layout can be simplified by excluding the applied filters and including an action to clear all of the filters within the sidebar itself.
+
+With this method, the filter values being applied to the dataset are already displayed and implied in the sidebar.
+
+**Ex 2.2: Excludes the applied filters when positioned in a sidebar**
+
+![Applied filters with clear all in the sidebar](/assets/patterns/filter-patterns/applied-filters-positioning-sidebar-clear-all.png)
```

Comment:
Notice: here the icon of "clear all" is a plus instead of a cross, probably you want to use the same as in the other examples